NuRadioReco/examples/RNO_data/run_reconstruction.py

```diff
@@ -0,0 +1,71 @@
+import NuRadioReco
+import matplotlib.pyplot as plt
+from NuRadioReco.modules.io.rno_g.readRNOGData import readRNOGData
+import pandas as pd
+import numpy as np
+from NuRadioReco.utilities import units
+from NuRadioReco.modules import channelBandPassFilter
+from NuRadioReco.detector import detector
+import datetime
+from NuRadioReco.modules import sphericalWaveFitter
+from NuRadioReco.modules import channelAddCableDelay
+
+""" An example to show how to read RNO-G data and how to perform simple reconstructions. The data used is pulser data and a simple (brute force, not optimized) spherical wave reconstruction is performed to obtain the pulser position """""
+
+""" Initiazize modules needed"""
+
+channelBandPassFilter = NuRadioReco.modules.channelBandPassFilter.channelBandPassFilter()
+sphericalWaveFitter = NuRadioReco.modules.sphericalWaveFitter.sphericalWaveFitter()
+channelAddCableDelay = NuRadioReco.modules.channelAddCableDelay.channelAddCableDelay()
+
+
+use_channels = [10, 0, 23]
+sphericalWaveFitter.begin(channel_ids = use_channels)
+
+""" Specify the detector. """
+
+det = detector.Detector(json_filename = "detector_description.json")
+det.update(datetime.datetime(2022, 10, 1))
+
+""" Get positions for the pulsers from the detector file as starting positions for the fit """
+
+station_id = 21
+pulser_id = 3  #Helper string C
+pulser_position = det.get_relative_position(station_id, pulser_id, mode = 'device')
+rel_pulser_position = [pulser_position[0], pulser_position[1], pulser_position[2]]
```

Comment:
isn't the return of get_relative_position already a numpy array? Probably one doesn't need this line

Reply:
True